version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
why netperg appears twice ?

On Mon, 29 Aug 2016 at 21:53 Muli Ben-Yehuda <notifications@github.com>
wrote:

> In version.sh
> <https://github.com/LightBitsLabs/netperf/pull/1#discussion_r76664940>:
>
> > @@ -0,0 +1,12 @@
> > +#!/bin/bash
>
> Other than this, ship it :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LightBitsLabs/netperf/pull/1/files/5d6b33211566959a3f63ac6c0dc80c37bf1bf4fd#r76664940>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQ5x6lUxF3TO9R_f-MJ1PR6yWSCmH7eAks5qkyqvgaJpZM4JviDQ>
> .
>